### ISSUE TITLE
Tensorflow2 support

### ISSUE THREAD

**Issue by mrunalsompura1995** [ybkscht/EfficientPose#50]
Hello Everyone.

I am trying to run EfficientPose on RTX A4000 GPU, it works fine with GTX series but with higher CUDA versions, I guess tensorflow 1.15 is not sufficient. It needs to be run with tensorflow2 to get advantage of computation power. 

If there is support for tensorflow2 or any pytorch implementation of EfficientPose it would be great help.

Thanks in advance.

**Comment by jusepv** [comment]
I had the same problem with RTX 3090. I modified it to version2, but i think there's a problem with training.

**Comment by Kaladin-Syl-WR** [comment]
Hi @jusepv 

Have you been able to get it working with Tensorflow 2? Specifically, the inference, as the training is not an issue.

**Comment by armando-genis** [comment]
HI. Is anyone got to make the code in tensorflow 2.x? . I made it in torch but somethings to refine
